### PR TITLE
Multiplayer

### PR DESCRIPTION
**Pull request by Taeir**
This pull request consists (solely) of the changes of #171, #172 and #173, which all have been reviewed already.
If you see any checkstyle/pmd/findbugs warnings that need to be fixed before merging into master, then please make comments. Otherwise, I think this is good to be merged :)

<feature

### CODE REVIEW

**Comment by Diocruel** [comment]
Everything has been reviewed, ready to go!
+1
<review

File: src/main/java/com/github/fishio/multiplayer/server/MultiplayerServerPlayingField.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	}
+	
+	/**
+	 * Creates a new playerfish for a client.
+	 * 
+	 * @return
+	 * 		a new PlayerFish without any keys registered.
+	 */
+	public PlayerFish createClientPlayer() {
+		Image sprite = Preloader.getImageOrLoad("sprites/fish/playerFish.png");
```
Comment (LMiljak) [inline src/main/java/com/github/fishio/multiplayer/server/MultiplayerServerPlayingField.java:83]: _Auto-generated comment_

```
The String literal "sprites/fish/playerFish.png" appears 6 times in this file; the first occurrence is on line 83
```

File: src/test/java/com/github/fishio/TestPlayingField.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import com.github.fishio.behaviours.VerticalBehaviour;
 import com.github.fishio.game.GameState;
 import com.github.fishio.gui.GuiTest;
```
Comment (LMiljak) [inline src/test/java/com/github/fishio/TestPlayingField.java:15]: _Auto-generated comment_

```
Avoid unused imports such as 'com.github.fishio.gui.GuiTest'
```

**Comment by LMiljak** [comment]
Those 2 comments I made above can be easily fixed. There were many warnings however about unused fields, empty if statements, but that's all TODO and will be fixed another time.

<feature

**Pull request merged**